dts/bindings/dac/renesas,ra-dac-global.yaml

```
@@ -10,16 +10,24 @@ include: [base.yaml]
 properties:
   has-internal-output:
     type: boolean
-    description: True if the SoC supports the internal output feature in the DAC HWIP
+    description: |
+      True if the SoC supports the "internal output" feature in the DAC HWIP.
```

nit: using "True..." terminology is a bit ackward given this can't really be false. 
```suggestion
      Set if the SoC supports the "internal output" feature in the DAC HWIP.
```

ditto for others

---

Thank you, I will update this in future DAC PR